dependencies/scons-config/sconsconfig/packages/PETSc.py

```diff
@@ -20,8 +20,7 @@ class PETSc(Package):
 
   def __init__(self, **kwargs):
     defaults = {
-      #'download_url': 'http://ftp.mcs.anl.gov/pub/petsc/release-snapshots/petsc-lite-3.7.6.tar.gz',
-      'download_url': 'http://ftp.mcs.anl.gov/pub/petsc/release-snapshots/petsc-lite-3.12.3.tar.gz',
+      'download_url': 'http://ftp.mcs.anl.gov/pub/petsc/release-snapshots/petsc-lite-3.15.5.tar.gz'
```

Comment:
just out of interest, is the version 3.15.5 a special one? The most recent petsc-lite snapshot seems to be [petsc-lite-3.19.6.tar.gz](https://ftp.mcs.anl.gov/pub/petsc/release-snapshots/petsc-lite-3.19.6.tar.gz)

Reply:
I chose this one because i noticed that when doing `sudo apt-get install petsc-dev` in ubuntu 20 it called to petsc 3.12 bur for ubuntu 22 it called for petsc 3.15.5